### PR TITLE
LPS-72795 Remove _onClickItem function once it is being called on A.Pagination

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

**Comment by pedroqueiroz94** [comment]
ci:retest

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pagination.js [hunk 1]
```diff
@@ -74,6 +78,39 @@ AUI.add(
 						instance._renderItemsUI(instance.get('total'));
 					},
 
+					_onClickItem: function(event) {
+						var instance = this;
+
+						var	item = event.currentTarget;
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pagination.js]: @alinedoleron @pedroqueiroz94 This indentation seems strange. Do we run formatSource? Thanks!

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js [hunk 2]
```diff
@@ -316,6 +367,27 @@ AUI.add(
 						switchModeNode.toggle(event.newVal > 1);
 					},
 
+					_afterPaginationPageChange: function() {
+						var instance = this;
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js]: @alinedoleron @pedroqueiroz94 I recently changed and merged this file. It seems my changes are not here. Did you guys rebase with master? Thanks!

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js [hunk 3]
```diff
@@ -733,13 +861,38 @@ AUI.add(
...
+
 						instance._getPopover().hide();
 
 						if (instance.get('mode') === 'pagination') {
+							if (pagination.get('successPage') && paginationSelected === totalPages + 1) {
+								wizard._activeSuccessPage = true;
+								wizard.set('selected', totalPages);
+								wizard.set('items', items);
+								pagination.set('page', totalPages + 1);
+
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js]: @alinedoleron @pedroqueiroz94 Space not needed

File: dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js [hunk 4]
```diff
@@ -733,13 +861,38 @@ AUI.add(
...
+							}
 							instance.set('mode', 'wizard');
+							wizard.set('selected', paginationSelected - 1);
+
```
Comment (leoadb) [inline dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_pages_manager.js]: @alinedoleron @pedroqueiroz94 Space not needed

**Comment by pedroqueiroz94** [comment]
ci:retest

**Pull request merged**